tests/test_mcp_toolset.py

```
@@ -49,10 +48,10 @@ def test_mcpserver_unreachable():
     assert (
         False,
         "Failed to load mcp server test_mcp http://0.0.0.0:3009/sse ('unhandled errors in a TaskGroup', [ConnectError('All connection attempts failed')])",
-    ) == mcp_toolset.init_server_tools(config=None)
+    ) == mcp_toolset.init_server_tools()
```

Ruff (0.11.9)

Yoda condition detected

(SIM300)